packages/auth/demo/src/index.js

```diff
@@ -277,7 +277,7 @@ function onAuthError(error) {
     // Handle second factor sign-in.
     handleMultiFactorSignIn(getMultiFactorResolver(auth, error));
   } else {
-    alertError('Error: ' + error.code);
+    alertError('Error: ' + error.code + 'user email : ' + error.customData.email);
```

Comment:
nit, put a space in front of "user email" so that it doesn't print right next to the error code

Reply:
Removed this line.